### PR TITLE
Add missing machine config for external open shift clusters. for hpp storage

### PR DESCRIPTION
**Pull request by awels**
Signed-off-by: Alexander Wels <awels@redhat.com>

**What this PR does / why we need it**:
We had a call to deploy the machine config, but never added it to the repo. This PR fixes that.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by maya-r** [comment]
How come this is different from https://github.com/kubevirt/hostpath-provisioner-operator/blob/master/contrib/machineconfig-selinux-hpp.yaml ?
is the master part not necessary?

**Comment by awels** [comment]
So the master part is for if you want the hpp to also run on masters, by default the expectation is that hpp runs only on workers as that is where the VMs will be running.

**Comment by mhenriks** [comment]
Did not work for me with crc 1.20.0

**Comment by awels** [comment]
Actually now that we thought about it a bit more, it makes more sense to update the script to point to the yaml in the hpp repo, instead of duplicating it here.

File: cluster-sync/external/provider.sh [hunk 1]
```diff
@@ -21,7 +21,7 @@ function up() {
 
 function configure_storage() {
   if [[ $KUBEVIRT_STORAGE == "hpp" ]] ; then
-    _kubectl apply -f ./cluster-sync/external/resources/machineconfig-worker.yaml
+    _kubectl apply -f https://github.com/kubevirt/hostpath-provisioner-operator/blob/master/contrib/machineconfig-selinux-hpp.yaml
```
Comment (mhenriks) [inline cluster-sync/external/provider.sh:24]: Maybe we should check machinconfig crd installed?  SOmething similar to:

https://github.com/kubevirt/containerized-data-importer/blob/master/cluster-sync/ephemeral_provider.sh#L56-L58

**Pull request closed without merging**